components/data-table/__examples__/advanced.jsx

```diff
@@ -77,8 +77,9 @@ const Example = createReactClass({
 		};
 	},
 
-	handleChanged (selection) {
-		this.setState({ selection });
+	handleChanged (event, data) {
+		this.setState({ selection: data.selection });
+		console.log(event, data);
```

Comment:
Are we allowing these in this codebase? (I always forget which project has which practices 😃)

Reply:
nm, makes sense here in the example if one wanted to see how stuff is working.

Reply:
Yep, here is where they are allowed to pass lint. https://github.com/salesforce/design-system-react/blob/master/.eslintrc.json#L22